src/irm.cpp

```diff
@@ -110,35 +104,23 @@ usage() {
         " ",
         "Remove one or more data-object or collection from iRODS space. By default, ",
```

Comment:
"data-object or collection" -> "data objects and/or collections"

I see use of `data-object` and `data object`.

Please update the help text so that only one form is used.

Reply:
Will do

Reply:
Done. Also reworded a couple of the statements for clarification.